released/packages/coq-menhirlib/coq-menhirlib.20210310/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "A support library for verified Coq parsers produced by Menhir"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "Jacques-Henri Jourdan <jacques-henri.jourdan@lri.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/coq-menhirlib"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/menhir.git"
+bug-reports: "jacques-henri.jourdan@lri.fr"
```

Comment:
```suggestion
bug-reports: "https://gitlab.inria.fr/fpottier/menhir/-/issues"
```

If you don't mind I'll click to commit this change for you and merge the PR

Reply:
Fine with me, thanks.